vkubelet/vkubelet.go

```diff
@@ -189,7 +138,6 @@ func (s *Server) registerNode(ctx context.Context) error {
 				Architecture:    "amd64",
 				KubeletVersion:  "v1.11.2",
 			},
-			Capacity:        s.provider.Capacity(ctx),
```

Comment:
why Capacity is not set?

Reply:
Good question. Don't remember even touching this. Nice catch.